apps/src/music/views/TimelineTrackEvents.jsx

```diff
@@ -5,6 +5,7 @@ import moduleStyles from './timeline.module.scss';
 import TimelineElement from './TimelineElement';
 
 const TimelineTrackEvents = ({
+  currentMeasure,
```

Comment:
We can just grab this from `playerUtils.getCurrentMeasureInternal()` right? No need to pass through as props?

Reply:
The one risk is that this progresses constantly since it uses Web Audio, and so we could get minor inconsistencies when a render spans a transition between which sounds are playing.

Reply:
As discussed offline, decided to pass the playhead position into `Timeline` which can then pass the same value down to everything it renders.